### PR TITLE
Update formatting to make request/responses easier to read

### PR DESCRIPTION
**Pull request by colinhumber**

### CODE REVIEW

File: Sources/Utils/RequestModelBeautifier.swift [hunk 1]
```diff
@@ -102,20 +102,11 @@ extension Dictionary {
...
-            formattedJSON = self
-        }
-        
-        return formattedJSON
+        guard let stringData = self.data(using: .utf8),
+            let object = try? JSONSerialization.jsonObject(with: stringData, options: []),
+            let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]),
+            let formattedJSON = String(data: jsonData, encoding: .utf8) else { return nil }
+
+        return formattedJSON.replacingOccurrences(of: "\\", with: "")
```
Comment (pmusolino) [inline Sources/Utils/RequestModelBeautifier.swift]: This is a little bit weird... if a string contains `\\` occurrences, you replace it. I'm not sure it's the best way to do it.

Comment (colinhumber) [reply Sources/Utils/RequestModelBeautifier.swift]: Oh good call. I did this a little haphazardly 😬. Just pushed an update that correctly replaces escaped forward slashes (\\/) with a normal slash (/).

**Comment by colinhumber** [comment]
The `prettyPrintedJSON` method that was there before was generating some incorrect JSON. For example, in the `/photos` response there were some commas missing.

```
[
{
  "thumbnailUrl" : "https:\/\/via.placeholder.com\/150\/92c952",
  "url" : "https:\/\/via.placeholder.com\/600\/92c952",
  "title" : "accusamus beatae ad facilis cum similique qui sunt",
  "id" : 1,
  "albumId" : 1
}{ <-- here
  "albumId" : 1,
  "id" : 2,
  "title" : "reprehenderit est deserunt velit ipsam",
  "thumbnailUrl" : "https:\/\/via.placeholder.com\/150\/771796",
  "url" : "https:\/\/via.placeholder.com\/600\/771796"
}{ <-- here
  "id" : 3,
  "url" : "https:\/\/via.placeholder.com\/600\/24f355",
  "albumId" : 1,
  "thumbnailUrl" : "https:\/\/via.placeholder.com\/150\/24f355",
  "title" : "officia porro iure quia iusto qui ipsa ut modi"
}....
```

I switched it to let the system do the pretty printing as the `Dictionary.prettyPrintedJSON` was doing.

**Comment by colinhumber** [comment]
@pmusolino Btw thanks for all the work on this library!

**Comment by pmusolino** [comment]
Thank you very much @colinhumber for clarifying this particular!

**Pull request merged**